MiniDumper/MiniDumper.cs

```diff
@@ -178,14 +200,85 @@ public void PrintDebugString(OUTPUT_DEBUG_STRING_INFO outputDbgStrInfo)
             }
         }
 
+        public void MemoryCommitThreshold(uint? commitThreshold, uint? commitDrops, int NumberOfDumps, ref bool isTimerSet)
+        {
+            isTimerSet = true;
+            CommitThreshold = commitThreshold * 1048576;
+            CommitDrops = commitDrops * 1048576;
+
+            hProcess = ProcessNativeMethod.OpenProcess(ProcessAccessFlags.QueryInformation, false, pid);
+            if (hProcess.IsInvalid)
+                throw new ArgumentException(String.Format("Unable to open process {0}, error {x:8}", pid, Marshal.GetLastWin32Error()));
```

Comment:
The "{x:8}" format string is wrong - I guess you meant "{1:x8}". There are two more places with the same issue - please check VS warnings.